In this repository we deposited the Supplementary data/script used in study "Genome-wide inference of Camponotus floridanus protein-protein interaction network using homologous mapping and interacting domain profile pairs".

## Supplementary data and material
Additional data and material can be found [here](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/master/Supplementary_materials.zip) in a single zip file. Sequence alighment of Single Copy Orthologs can be obtained from [here](https://funginet.hki-jena.de/data_files/76)

#### [Additional data file 1](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/supplementary_data/Additional%20data%20file%201.pdf):

Clusters of orthologues identified by OrthoMCL (inflation index: 1.5). Abbreviations - abra : *Aspergillus brasiliensis*, acar : *A. carbonarius*, acla : *A. clavatus*, afis : *A. fischeri*, afla : *A. flavus*, afum : *A. fumigatus*, agla : *A. glaucus*, akaw : *A. kawachii*, alue : *A. luchuensis*, anid : *A. nidulans*, anig : *A. niger*, aora : *A. oryzae*, asyd : *A. sydowii*, ater : *A. terreus*, atub : *A. tubingenesis*, aver : *A. versicolor*, awen : *A. wentii*, azon : *A. zonatus*.

#### [Additional data file 2](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/supplementary_data/Additional%20data%20file%202.pdf):

Core genome of Aspergillus. The file consists of all the clusters containing at least one protein from all the analysed 18 species. Abbreviations - abra : *Aspergillus brasiliensis*, acar : *A. carbonarius*, acla : *A. clavatus*, afis : *A. fischeri*, afla : *A. flavus*, afum : *A. fumigatus*, agla : *A. glaucus*, akaw : *A. kawachii*, alue : *A. luchuensis*, anid : *A. nidulans*, anig : *A. niger*, aora : *A. oryzae*, asyd : *A. sydowii*, ater : *A. terreus*, atub : *A. tubingenesis*, aver : *A. versicolor*, awen : *A. wentii*, azon : *A. zonatus*.

#### [Additional data file 3](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/supplementary_data/Additional%20data%20file%203.pdf): 

*A. fumigatus* and *A. fischeri* orthology analysis (Inparanoid output).

#### [Additional file 1](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/supplementary_data/Additional%20file%201.xlsx) –

Table S1. Basic genomic information of all eighteen aspergilli

Table S2. GO over-representation analysis of single copy orthologs in the three GO categories (i.e. ‘Biological Processes’, BP; ‘Molecular Function’, MF; ‘Cellular Component’, CC) 

Table S3. Quantitative summary of orthology assignments

Table S4. Recombination breakpoints in alignments

Table S5. GO over-representation analysis of recombination breakpoint containing clusters in the three GO categories (i.e. ‘Biological Processes’, BP; ‘Molecular Function’, MF; ‘Cellular Component’, CC)

Table S6. Positively selected genes and KOG annotations

Table S7. Positively selected genes and their expression in hypoxia conditions

Table S8. *A. fumigatus* positively selected hub proteins

Table S9. Degree of *A. fumigatus* positively selected proteins involved in host interactions

Table S10. Pathway overrepresentation of host proteins interacting with *A. fumigatus* positively selected proteins

Table S11. Gene Ontology (GO) overrepresentation of host proteins interacting with *A. fumigatus* positively selected proteins

Table S12. Positively selected genes identified in *A. fischeri* 

#### [Additional file 2: Supplementary figures](https://github.com/ShishirGupta-Wu/aspergillus_ps/blob/supplementary_data/Additional%20file%202.pdf) -

Figure S1: Proteins-protein interactions (PPIs) map. PPIs between *A. fumigatus* proteins coded by positively selected genes (PSGs) and human proteins. Orange - *A. fumigatus* proteins and blue - human proteins. 

Figure S2: *A. fumigatus* and *A. fischeri* genome synteny map. Image was generated using the Mauve genome alignment tool. Upper and lower lines of the genomes correspond to *A. fumigatus* and *A. fischeri* respectively. Red vertical bars indicate concatenated chromosomal boundaries. Color-coded syntenic blocks indicate conserved segments (LCBs; Locally Collinear Blocks) identified by Mauve (minimum LCB weight = 999). Plots of sequence similarity are shown within each syntenic block. Regions with no color indicate no detectable homology between the two genomes with the settings used in Mauve. Unmatched regions (white area) within an LCBs indicate the presence of strain-specific sequence. The connecting lines between blocks indicate the location of each block in two genomes. Homologous regions with possible rearrangements are shown in first row.
